index.html

```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
+
```

Comment:
- [ ] As you go with the Microvers program you will be building amazing projects, a repo that contains many branches 😎 and a really specific branch name that conveys an action or a feature you trying to build in this Milestone will just make the PR so meaningful and it will also help you navigate easily through your GitHub repo branches.

So kindly suggest renaming the current branch to something more meaningful such as `setup-hello-microvese` or `feature/implement-hello-microvers`.

- Please consider checking how to [Rename a branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) to easily do so.